packages/labs/tsserver-plugin/example/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    // This matches the package name given in this package.json
+    "plugins": [
+      {
+        "name": "@lit-labs/tsserver-plugin"
```

Comment:
This works? Cool

Reply:
yeah - I got this from the tsserver-plugin example repo link'ed from the TS wiki. It's nice. The only problem is that the circular symlink required blows up eslint right now so you have to remove the example's node_modules/ folder to lint.